components/constants/max-age.js

```diff
@@ -0,0 +1 @@
+export const MAX_AGE = 43200;
```

Comment:
Maybe make this a bit more descriptive? Max age of _what_?

Reply:
Yes Sure, I will make it CACHE_MAX_AGE